src/GafferSceneUI/CropWindowTool.cpp

```diff
@@ -459,6 +464,49 @@ void CropWindowTool::plugDirtied( const Gaffer::Plug *plug )
 	}
 }
 
+void CropWindowTool::nodeMetadataChanged( IECore::InternedString key, const Gaffer::Node* node )
+{
+	if( !m_cropWindowPlug || m_overlayDirty )
```

Comment:
Does the dirtiness of the overlay really mean we can early-out here? A viewport change can dirty the overlay without setting `m_needCropWindowPlugSearch`.

Reply:
Which lead me to find: https://github.com/GafferHQ/gaffer/pull/3239/commits/32f6337344b5e0ef29afc660bc1f2c7e30e1ccb5

Fixed in https://github.com/GafferHQ/gaffer/pull/3239/commits/558040aa1361bf1df48a471a037f3655bff30c97